script/deploy/4_VerifyOFT.s.sol

```diff
@@ -137,33 +171,112 @@ contract VerifyOFT is BaseScript, BatchScript {
 
             ChainDeployment memory chainDeployment = findChainDeployment(chainId);
 
-            (,, uint256 limit, uint256 window) = RateLimiter(payable(currentDeployment.oftAdapter)).rateLimits(eid);
-            if (limit != baseInput.rateLimitConfig.limit || window != baseInput.rateLimitConfig.window) {
-                needsUpdate = true;
-                newRateLimitConfigs.push(
-                    RateLimiter.RateLimitConfig(
-                        eid, baseInput.rateLimitConfig.limit, baseInput.rateLimitConfig.window
-                    )
-                );
+            {
+                // verify rate limits
```

Comment:
no changes in exsiting verification steps, except its inside `{}` to ensure we don't have "stack too deep" error